android/src/test/java/com/fidelreactlibrary/FidelSetupAdapterTests.java

```diff
@@ -790,6 +792,45 @@ public void test_IfDoesHaveProgramTypeKey_ShouldSetTheValueProvidedByTheProgramT
         assertEquals(ProgramType.TRANSACTION_STREAM, Fidel.programType);
     }
 
+    @Test
+    public void test_IfDoesNotHaveEnableCardScannerKey_DoNotSetThisPropertyForFidel() {
+        ReadableMapStub map = ReadableMapStub.withoutOptionsKey(FidelSetupKeys.Options.ENABLE_CARD_SCANNER);
+        ReadableMapStub optionsMap = (ReadableMapStub) map.mapsForKeysToReturn.get(FidelSetupKeys.OPTIONS.jsName());
+        assertNotNull(optionsMap);
+
+        Fidel.enableCardScanner = true;
```

Comment:
Why are we doing this and then checking that's true? 

Wouldn't this be enough to test what we have in the description? 
```
assertFalse(optionsMap.keyNamesAskedValueFor.contains(FidelSetupKeys.Options.ENABLE_CARD_SCANNER.jsName()));
```

Reply:
I copied it from another test but basically we want to check the property is set correctly. The optionsMap is a stub object created for testing purposes.

Reply:
For the following tests, it makes sense to assert the `Fidel.enableCardScanner`, but for this one, I am not sure it does. But not a blocker on my side 😄 I let you decide!

Reply:
Got it 😄